test/e2e/flowaggregator_test.go

```diff
@@ -220,6 +224,7 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	// Iterate over recordSlices and build some results to test with expected results
 	recordSlices := getRecordsFromOutput(collectorOutput)
 	dataRecordsCount := 0
+	var octetTotalCount float64
```

Comment:
`octetTotalCount` should be `uint64` format

Reply:
Thanks, will address soon.